app/assets/stylesheets/common/base/explain-reviewable.scss

```diff
@@ -31,7 +31,7 @@
     color: var(--primary-medium);
   }
 
-  .op:last-of-type {
+  .op:first-of-type {
```

Comment:
Confirmed this is [supported in all browsers](https://caniuse.com/?search=first-of-type).